W2/hw nursultan moldataev phiest34

phiest34 opened the pull request.

phiest34 committed:
first hometask

phiest34 committed:
first hometask

erkhabibullina commented on Week1/app/src/main/java/com/example/android/diceroller/MainActivity.kt:
```diff
@@ -4,23 +4,39 @@ import android.os.Bundle
 
     // todo (06) Создайте функцию rollDice. Задания 7-9 выполнить внутри тела функции rollDice()
+    fun rollDice() {
+        val randomInt:Int = Random().nextInt(6) + 1
```
Тут необязательно указывать тип, Kotlin сам поймёт, что ты имеешь в виду `Int`

erkhabibullina reviewed (commented):
Очень хорошо, молодец)))

phiest34 committed:
merged

# Conflicts:
#	.idea/codeStyles/Project.xml
#	Week1/.idea/.gitignore
#	Week1/.idea/codeStyles/Project.xml
#	Week1/app/src/main/java/com/example/android/diceroller/MainActivity.kt
#	Week1/local.properties

phiest34 committed:
second home task
